source/ProcessManager.Orchestrations/Processes/BRS_021/ForwardMeteredData/V1/Thingies/MeteringPointMasterDataProvider.cs

```diff
@@ -0,0 +1,144 @@
+﻿// Copyright 2020 Energinet DataHub A/S
+//
+// Licensed under the Apache License, Version 2.0 (the "License2");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Diagnostics.CodeAnalysis;
+using Energinet.DataHub.ElectricityMarket.Integration;
+using Energinet.DataHub.ElectricityMarket.Integration.Models.MasterData;
+using Energinet.DataHub.ProcessManager.Abstractions.Core.ValueObjects;
+using Energinet.DataHub.ProcessManager.Core.Domain.OrchestrationInstance;
+using Energinet.DataHub.ProcessManager.Orchestrations.Extensions.Mapper;
+using Energinet.DataHub.ProcessManager.Orchestrations.Processes.BRS_021.ForwardMeteredData.V1.Extensions;
+using Energinet.DataHub.ProcessManager.Orchestrations.Processes.BRS_021.ForwardMeteredData.V1.Mapper;
+using Microsoft.Extensions.Logging;
+using NodaTime;
+using PMGridAreaCode =
+    Energinet.DataHub.ProcessManager.Orchestrations.Processes.BRS_021.ForwardMeteredData.V1.Model.GridAreaCode;
+using PMMeteringPointMasterData =
+    Energinet.DataHub.ProcessManager.Orchestrations.Processes.BRS_021.ForwardMeteredData.V1.Model.
+    MeteringPointMasterData;
+using PMResolution = Energinet.DataHub.ProcessManager.Components.Abstractions.ValueObjects.Resolution;
+
+namespace Energinet.DataHub.ProcessManager.Orchestrations.Processes.BRS_021.ForwardMeteredData.V1.Thingies;
+
+[SuppressMessage(
+    "StyleCop.CSharp.ReadabilityRules",
+    "SA1118:Parameter should not span multiple lines",
+    Justification = "Readability")]
+public class MeteringPointMasterDataProvider(
+    IElectricityMarketViews electricityMarketViews,
+    ILogger<MeteringPointMasterDataProvider> logger)
+{
+    private readonly IElectricityMarketViews _electricityMarketViews = electricityMarketViews;
+    private readonly ILogger<MeteringPointMasterDataProvider> _logger = logger;
+
+    internal async Task<IReadOnlyCollection<PMMeteringPointMasterData>> GetAndConvertMasterData(
```

Comment:
```suggestion
    internal async Task<IReadOnlyCollection<PMMeteringPointMasterData>> GetMasterData(
```

The consumer doesn't need to know that the data is converted, it just needs to know that it retrieves the master data. If the data is returned directly from an API or we map should just be an implementation detail that only lives inside `MeteringPointMasterDataProvider`